tests/test_gamma.py

```diff
@@ -26,10 +26,8 @@ def test_sd():
 )
 def test_from_quantiles(x0_x1):
     x0, x1 = x0_x1
-    if x0 == x1:
-        return
-    if x0 > x1:
-        x1, x0 = x0, x1
+    assume(abs(x1 - x0) > 1e-6)
+    assume(x0 < x1)
```

Comment:
Since we ordered x0 and x1 just above, this might be unnecessary.

Reply:
You mean the abs?

Reply:
Sorry, it was my mistake. That is fine.